full_build.sh

```diff
@@ -28,13 +28,33 @@ git checkout composer.json
 rm -rf rector-build/packages-tests rector-build/rules-tests rector-build/tests rector-build/bin/generate-changelog.php rector-build/bin/validate-phpstan-version.php
 
 sh build/downgrade-rector.sh rector-build
-sh build/build-rector-scoped.sh rector-build rector-prefixed-downgraded
 
-cd rector-prefixed-downgraded
+cd rector-build
+
+# avoid syntax error in php 7.1 and 7.2
+rm rector.php
+
 cp ../build/target-repository/bootstrap.php .
 cp ../preload.php .
-bin/rector list --ansi
-bin/rector process vendor/symfony/string/Slugger/ --dry-run
+
+# Update to your local php 7.1 path
+/opt/homebrew/Cellar/php@7.1/7.1.33_4/bin/php bin/rector list --ansi
```

Comment:
That would not work for me, as I don't use homebrew. This should remain generic as before.

Reply:
I think the script need to read ENV variable then, eg:

```php
export PHP_LOCATION_BIN=/my/php/bin/php ./full_build.sh
```

what do you think?

Reply:
That could work. With keeping default path as in original script.

Reply:
I will try.